pkgs/top-level/all-packages.nix

```
@@ -4996,7 +4996,7 @@ with pkgs;
 
   datovka = libsForQt5.callPackage ../applications/networking/datovka { };
 
-  dave = callPackage ../servers/http/dave { };
+  dave = lib.warn "dave is unmaintained and will be removed in a future release" (callPackage ../servers/http/dave { });
```

Ofborg does not like it.

```suggestion
  dave = callPackage ../servers/http/dave { };
```

---

Is there really no way to warn a package is deprecated?

---

I can imagine a way: something like "if version is at least 24.01 then append the lib.warn else leave it as is".

Ugly and a bit impure, but it guarantees that everyone will receive the dreadful warning next year :)

---

What about `aliases.nix`, will that case the same problem with ofborg?

---

Yes it will.

Anything under aliases.nix is treated badly by ofborg.